templates/core/build.html

```diff
@@ -31,239 +31,226 @@ <h1>Your Ubuntu image is building</h1>
-          <button type="submit" class="p-search-box__button" alt="search"><i class="p-icon--search"></i></button>
-        </form>
-        <div class="p-card has-limited-height">
-          <div class="js-snap-results"></div>
-        </div>
-      </div>
-      <div class="col-6">
-        <h4>Your chosen snaps:</h4>
-        <div class="p-card has-limited-height--large">
-          <div class="js-preinstalled-snaps-list"></div>
-        </div>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h4>Available snaps:</h4>
+      <form class="p-search-box js-snap-search">
+        <input type="search" class="p-search-box__input" name="search" required="">
+        <button type="reset" class="p-search-box__reset" alt="reset"><i class="p-icon--close"></i></button>
+        <button type="submit" class="p-search-box__button" alt="search"><i class="p-icon--search"></i></button>
+      </form>
+      <div class="p-card has-limited-height">
+        <div class="js-snap-results"></div>
       </div>
     </div>
-    <div class="row">
-      <div class="col-8">
-        <p>Got an app that isn’t in the Snap Store? Learn how to <a href="https://snapcraft.io/">publish it as a snap</a>.</p>
+    <div class="col-6">
+      <h4>Your chosen snaps:</h4>
+      <div class="p-card has-limited-height--large">
+        <div class="js-preinstalled-snaps-list"></div>
       </div>
     </div>
-  </section>
+  </div>
+</section>
+<section class="p-strip is-shallow u-no-padding--top">
+  <div class="row">
+    <div class="col-8">
+      <p>Got an app that isn’t in the Snap Store? Learn how to <a href="https://snapcraft.io/">publish it as a snap</a>.</p>
+    </div>
+  </div>
+</section>
 
-  <section class="p-strip--light is-shallow">
-    <div class="row">
-      <div class="col-8">
-        <form method="POST" id="build-form">
-          <p>We’ll email you at {{ openid.email }} once the build is ready.</p>
-          <p>
-            <input class="mktoField" value="yes" id="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
-            <label class="mktoLabel mktoHasWidth" for="canonicalUpdatesOptIn">I agree to receive information about Canonical’s products and services.</label>
-          </p>
-          <input type="hidden" name="board" value="" />
-          <input type="hidden" name="system" value="" />
-          <input type="hidden" name="snaps" value="" />
-          <input type="hidden" name="FullName" value="{{ openid.fullname }}" />
-          <input type="hidden" name="Email" value="{{ openid.email }}" />
-          <p><button class="p-button--positive js-build-button">Build image</button></p>
-        </form>
-      </div>
+<section class="p-strip--light is-shallow  {% if not openid %}u-disable{% endif %}">
+  <div class="row">
+    <div class="col-8">
+      <form method="POST" id="build-form">
+        <p>We’ll email you at {{ openid.email }} once the build is ready.</p>
```

Comment:
I don't think the text makes too much sense if there is no emali.

Reply:
When did you see it without a email?

Reply:
Yep, in #6695 I said “It’s a bit unpolished that when signed out, we say ‘We’ll email you at once the build is ready.’ Could we drop the ‘at’ in that case?’”

Reply:
@anthonydillon it's at the bottom of the page.

Reply:
Ah, I see. I have hidden the message if there is no email.